webapp/src/actions.js

```diff
@@ -4,6 +4,20 @@ import {id as pluginId} from './manifest';
 import {STATUS_CHANGE, OPEN_ROOT_MODAL, CLOSE_ROOT_MODAL} from './action_types';
 
 export const openRootModal = () => (dispatch) => {
+    dispatch({
```

Comment:
Nit: could we have this call `openRootModalWithId('')` to avoid code duplication?

Reply:
I agree, though I haven't found a clean way to do it such that dispatch is passed along.  I may be missing something.
```javascript
dispatch(openRootModalWithSubmenuName(''));   //doesn't work either
```

Thoughts on the cleanest way to accomplish this?

Reply:
Opposed to leaving this in @marianunez ?  I am open to suggestions on changes, though don't have a better solution currently.

Reply:
@scottleedavis I suggest keeping the original `openRootModal` and having the optional text parameter like:

```suggestion
    export const openRootModal = (subMenuText = '') => (dispatch) => {
    dispatch({
        type: SUBMENU,
        subMenu: subMenuText,
    });
    dispatch({
        type: OPEN_ROOT_MODAL,
    });
};
```

I tried it locally and seems to work fine for me. You can ping me in the community server if you want to talk it over.

Reply:
This worked great, thank you @marianunez